js-src/compat/plugins/mondialrelay/checkout-mondialrelay.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Checkout scripts for: Mondial Relay - WordPress (by Kasutan).
+ */
+
+(function (root, factory) {
+	if ( typeof define === 'function' && define.amd ) {
+		define([], factory(root));
+	} else if ( typeof exports === 'object' ) {
+		module.exports = factory(root);
+	} else {
+		root.CheckoutMondialRelay = factory(root);
+	}
+})(typeof global !== 'undefined' ? global : this.window || this.global, function (root) {
+
+	'use strict';
+
+	var $ = jQuery;
+	var _hasJQuery = ( $ != null );
+
+	var _hasInitialized = false;
+	var _publicMethods = {};
+	var _settings = {
+		buttonSelector: '#delivery_point_chosen',
+	};
+
+
+
+	/**
+	 * METHODS
+	 */
+
+
+
+	/**
+	 * Update checkout fields.
+	 */
+	var updateCheckout = function() {
+		// Create update checkout event
+		var event = new Event( 'update_checkout' );
+
+		// Trigger event
+		document.body.dispatchEvent( event );
+	}
+
+
+
+	/**
+	 * Initialize component and set related handlers.
+	 */
+	_publicMethods.init = function() {
+		if ( _hasInitialized ) return;
+
+		// Update checkout fields on button click
+		if ( _hasJQuery ) {
+			$( _settings.buttonSelector ).on( 'click', updateCheckout );
```

Comment:
- Although we ultimately trigger a jQuery event when the button is clicked, the `click` event does not need jQuery and it would be more appropriate to use the captured event handler with the function `handleClick` detecting the element which was clicked and routing to the associated handling function.

Reply:
@vlad-dedikov, you missed making this change.